paddle/fluid/imperative/engine.h

```diff
@@ -51,6 +51,12 @@ class Engine {
     grad_ops_[op] = std::move(op_shared);
   }
 
+  const std::unordered_set<VarBase*>& GradVars() const { return grad_vars_; }
```

Comment:
GradVars never be called

Reply:
Thanks, I will add usage it in `test_tracer.cc ` to check `grad_vars_ ` not empty.